app/javascript/controllers/transition_controller.js

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus'
+
+// Example:
+//
+// <div data-controller="transition" data-transition-toggle-class="!hidden">
+//   <div id="element-that-shows-and-hides" class="block" data-transition-target="me"></div>
+//   <a href="#" data-action="transition#toggleClass">click to toggle</a>
+// </div>
+//
+// Every element that is of target = me will have the class toggled.
+
+export default class extends Controller {
+  static classes = [ "toggle" ]
+  static targets = [ "me" ]
+  static values = {
+    afterTimeout: Number
+  }
+
+  connect() {
+    console.log(`connect`)
```

Comment:
❌ 

The other day I turned on the stimulus debug mode in application.js and it emits this message automatically. It would probably cover your logs on line 25/26 too. It's _verbose_ though because it turns on debug statements for every controller and action.